packages/contact-center/station-login/src/helper.ts

```diff
@@ -72,12 +72,14 @@ export const useStationLogin = (props: UseStationLoginProps) => {
     cc.stationLogin({teamId: team, loginOption: deviceType, dialNumber: dialNumber})
       .then((res: StationLoginSuccess) => {
         setLoginSuccess(res);
+        setLoginFailure(undefined);
       })
       .catch((error: Error) => {
         logger.error(`Error logging in: ${error}`, {
           module: 'widget-station-login#helper.ts',
           method: 'login',
         });
+        setLoginSuccess(undefined);
```

Comment:
Could you please update the tests for this? Just to ensure we clear loginFailure on success and loginSuccess on failure?